model/Security/Classes/CvssV3VulnAssessmentRelationship.md

```diff
@@ -0,0 +1,75 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# CvssV3VulnAssessmentRelationship
+
+## Summary
+
+Provides a CVSS version 3.x assessment for a vulnerability.
+
+## Description
+
+A CvssV2VulnAssessmentRelationship relationship describes the characteristics and impact of a vulnerability using version 3.1 of the Common Vulnerability Scoring System (CVSS) as defined on [https://www.first.org/cvss/v3.1/specification-document](https://www.first.org/cvss/v3.1/specification-document). 
+
+## Metadata
+
+- name: CvssV3VulnAssessmentRelationship
+- SubclassOf: /Security/VulnAssessmentRelationship
+- Instantiability: Concrete
+
+## Properties
+
+- score
+  - type: xsd:string
+  - minCount: 0
+  - maxCount: 1
+- severity
+  - type: CvssV3SeverityType
+  - minCount: 1
+  - maxCount: 1
+- vector
+  - type: xsd:string
+  - minCount: 0
+  - maxCount: 1
+
+## Syntax
+
+=== "JSON"
+
+    ```json
+    {
+      "@type": "CvssV3VulnAssessmentRelationship",
+      "@id": "urn:spdx.dev:cvssv3-cve-2020-28498",
+      "relationshipType": "hasCvssV3AssessmentFor",
+      "severity": "medium",
+      "score": "6.8",
+      "vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:C/C:H/I:N/A:N",
+      "from": "urn:spdx.dev:vuln-cve-2020-28498",
+      "to": "urn:product-acme-application-1.3",
+      "assessedElement": "urn:npm-elliptic-6.5.2",
+      "externalReferences": [
+        {
+          "@type": "ExternalReference",
+          "externalReferenceType": "securityAdvisory",
+          "locator": "https://nvd.nist.gov/vuln/detail/CVE-2020-28498"
+        },
+        {
+          "@type": "ExternalReference",
+          "externalReferenceType": "securityAdvisory",
+          "locator": "https://snyk.io/vuln/SNYK-JS-ELLIPTIC-1064899"
+        },
+        {
+          "@type": "ExternalReference",
+          "externalReferenceType": "securityFix",
+          "locator": "https://github.com/indutny/elliptic/commit/441b742"
+        }
+      ]
+    },
+    {
+      "@type": "Relationship",
+      "@id": "urn:spdx.dev:vulnAgentRel-1",
+      "relationshipType": "foundBy",
+      "from": "urn:spdx.dev:agent-snyk",
+      "to": ["urn:spdx.dev:cvssv3-cve-2020-28498"],
+      "startTime": "2021-03-08T16:06:50Z"
```

Comment:
Same comment from slack on this on. The to/from is backwards for the foundBy relationship. Should probably use CreationInfo on the CvssV3VulnAssessmentRelationship relatinship like we do for EpssVulnAssessmentRelationship example:

```
{
  "@type": "EpssVulnAssessmentRelationship",
  "@id": "urn:spdx.dev:epss-1",
  "relationshipType": "hasEpssAssessmentFor",
  "probability": 80,
  "from": "urn:spdx.dev:vuln-cve-2020-28498",
  "to": "urn:product-acme-application-1.3",
  "creationInfo": {
    "@type": "CreationInformation",
    "created": "2021-03-09T11:04:53Z",
    "createdBy": ["urn:spdx.dev:agent-jane-doe"]
  }
}
```

Reply:
@rnjudge  We can only use creationInfo to indicate who created it unless add new properties such as foundby, publishedBy, republished by to CreationInfo in Core, or we can create a VulnCreationInfo which inherits from CreationInfo with all the fields we need.

Reply:
I've flipped the direction in the example. We should ship with these relationships for now and reflect if we need the inverse equivalents or simple replace them.